### DISCUSSION TITLE
See not only "Type" links in deepcase insertion menu

### DISCUSSION THREAD

**Discussion by FreePhoenix888** [deep-foundation/Discussions#25] [General]
We have to figure out how to add ability to see not just types, but "sub-types". It is actually hard to understand which link is sub-type and which is an instance of that type, but not sub-type

One of the easiest solution - show all the types which can be used for insertion

What do you think?

**Comment by Romanxz** [comment]
IF there is such possibiblity, types inside selection window can be nested into each other as type tree:
like this
![image](https://github.com/deep-foundation/Discussions/assets/44348954/d82c3462-9153-400b-b3cc-a687b43c8c6d)
